src/mockhsm/object/objects.rs

```diff
@@ -8,22 +8,28 @@ use crate::{
     serialization::{deserialize, serialize},
     wrap, Algorithm, Capability, Domain,
 };
-use aes::cipher::consts::{U13, U8};
+use aes::cipher::consts::{U13, U16};
 use ccm::aead::{AeadInPlace, KeyInit};
 use std::collections::{btree_map::Iter as MapIter, BTreeMap as Map};
 
 /// AES-CCM with a 128-bit key
-pub(crate) type Aes128Ccm = ccm::Ccm<aes::Aes128, U8, U13>;
+pub(crate) type Aes128Ccm = ccm::Ccm<aes::Aes128, U16, U13>;
+
+/// AES-CCM with a 192-bit key
+pub(crate) type Aes192Ccm = ccm::Ccm<aes::Aes192, U16, U13>;
 
 /// AES-CCM with a 256-bit key
-pub(crate) type Aes256Ccm = ccm::Ccm<aes::Aes256, U8, U13>;
+pub(crate) type Aes256Ccm = ccm::Ccm<aes::Aes256, U16, U13>;
```

Comment:
The tag/mac size is specified as 16 bytes: https://developers.yubico.com/YubiHSM2/Commands/Wrap_Data.html

Reply:
(this would be a breaking change in the mockhsm)